packages/asset-swapper/src/quote_consumers/exchange_proxy_swap_quote_consumer.ts

```diff
@@ -567,19 +598,30 @@ export class ExchangeProxySwapQuoteConsumer implements SwapQuoteConsumerBase {
         for_loop: for (const [i, order] of quote.orders.entries()) {
             switch_statement: switch (order.source) {
                 case ERC20BridgeSource.Native:
-                    if (order.type !== FillQuoteTransformerOrderType.Rfq) {
+                    if (order.type !== (FillQuoteTransformerOrderType.Rfq || FillQuoteTransformerOrderType.Otc)) {
```

Comment:
this doesn't look right. 

should it be `order.type !== Rfq && order.type !== Otc`?